### PR TITLE
Feature/bocm 355 modify get appeal details for inspector

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

- Seeds database with Full Planning Appeals
- Creates flag for appeals due for booking 
- Fixes transitions for state machine and database saving for Full Planning Appeals

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by radum — APPROVED** [review]

File: apps/api/src/server/app/repositories/appeal.repository.js [hunk 1]
```diff
@@ -64,32 +65,26 @@ const appealRepository = (function () {
 			});
 		},
 		getById: function (
```
Comment (brpins) [inline apps/api/src/server/app/repositories/appeal.repository.js:67]: This is now essentially the same signature as `getByIdIncluding`, so we should drop one of them (probably drop getByIdIncluding). However, we should use the signature from`getByIdIncluding` as the include keys are automatically generated by prisma based on the relations defined in the schema

Comment (mayya-bond-kc) [reply apps/api/src/server/app/repositories/appeal.repository.js:67]: As I have mentioned before, I disagree. By defining the inclusions in the attributes themselves, we are leaking prisma code into the rest of the repo. Additionally, we are running the risk of repeating a lot of the same code in many places.

**Pull request merged**